src/org/zaproxy/zap/extension/pscanrulesBeta/ZapAddOn.xml

```diff
@@ -8,6 +8,7 @@
 	<changes>
 	<![CDATA[
 	Issue 2576: CSRFCountermeasures, remove unneeded Attack and Evidence messages.<br>
+	Issue 2574: Update attack field and description.<br>
```

Comment:
It could mention the scanner being changed (and say that attack was removed instead).